src/modules/interfaces.ts

```diff
@@ -464,9 +464,14 @@ export class ModuleBase {
   }
 
   init() {
+    this.loadBasics();
+    this.loadTypeORM();
+  }
+
+  loadBasics() {
     if (!this.dirname) {
       this.dirname =
-        path.dirname(callsite()?.[1]?.getFileName?.()) ??
+        path.dirname(callsite()?.[2]?.getFileName?.()) ??
```

Comment:
why this change?

Reply:
this looks at the stack trace and since I added another function call it changed